docs/pages/x/api/charts/heatmap-tooltip.json

```diff
@@ -0,0 +1,135 @@
+{
+  "props": {
+    "anchorEl": {
+      "type": {
+        "name": "union",
+        "description": "(props, propName) => {\n  if (props[propName] == null) {\n    return new Error(`Prop '${propName}' is required but wasn't specified`);\n  }\n  if (typeof props[propName] !== 'object' || props[propName].nodeType !== 1) {\n    return new Error(`Expected prop '${propName}' to be of type Element`);\n  }\n  return null;\n}<br>&#124;&nbsp;func<br>&#124;&nbsp;{ contextElement?: (props, propName) => {\n  if (props[propName] == null) {\n    return null;\n  }\n  if (typeof props[propName] !== 'object' || props[propName].nodeType !== 1) {\n    return new Error(`Expected prop '${propName}' to be of type Element`);\n  }\n  return null;\n}, getBoundingClientRect: func }"
```

Comment:
@mui/xcharts does it make sense to have such "detailed" descriptions? 🤔 
<img width="884" alt="Screenshot 2024-11-25 at 12 50 10" src="https://github.com/user-attachments/assets/18a72e05-2a41-44a1-abe0-35eb4e46bf8a">

Reply:
+1 to copy https://mui.com/material-ui/api/popper/#popper-prop-anchorEl

Reply:
#15625.